app/models/vm_or_template.rb

```diff
@@ -44,6 +44,7 @@ class VmOrTemplate < ApplicationRecord
     "amazon"      => "Amazon",
     "redhat"      => "RedHat",
     "openstack"   => "OpenStack",
+    "oracle"      => "Oracle",
```

Comment:
To clarify, this is Oracle Cloud.

Reply:
:+1: the `vendor-oracle.svg` is being picked up already
![Screenshot from 2021-02-24 14-40-44](https://user-images.githubusercontent.com/12851112/109056368-5527f080-76ae-11eb-8862-2a70994209eb.png)